app/assets/javascripts/miq_application.js

```diff
@@ -737,7 +737,12 @@ function miqAjax(url, serialize_fields, options) {
     complete: true,
   };
 
-  miqJqueryRequest(url, _.extend(defaultOptions, options || {}, { data: data }));
+  miqJqueryRequest(url, _.extend(defaultOptions, options || {}, { data: data }))
+    .catch(function(err){
+      add_flash(__("Error requesting data from server"), 'error');
+      console.log(err);
+      return Promise.reject(err)
```

Comment:
missing semicolon here and below.

also: can this be done in a more generic way?

Either inside `miqJqueryRequest` (we already have some `options` hash here) or in some new function that would call `miqJqueryReques`